### PR TITLE
[16.0][IMP] purchase_advance_payment: add dedicated menu for advance payments

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
image

### CODE REVIEW

**Review by DavidJForgeFlow — APPROVED** [review]
Functional review :+1: Code also LGTM!

File: purchase_advance_payment/views/account_payment_views.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <field name="name">account.payment.search</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_search" />
+        <field name="arch" type="xml">
+            <filter name="transfers_filter" position="after">
+                <separator />
+                <filter
+                    string="Vendor Advance Payments"
+                    name="vendor_adv_payment_filter"
+                    domain="[('purchase_id', '!=', False)]"
```
Comment (MiquelRForgeFlow) [inline purchase_advance_payment/views/account_payment_views.xml:14]:
```suggestion
                    domain="[('partner_type', '=', 'supplier'),('purchase_id', '!=', False)]"
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Code LGTM

**Review by HviorForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**